### ISSUE TITLE
[FR] Also give option to invalidate reported sentences.

### ISSUE THREAD

**Issue by HarikalarKutusu** [common-voice/common-voice#4181]
**Is your feature request related to a problem? Please describe.**

When validating a recording we can have a situation where there is a typo, bad grammar in the sentence etc, but the speaker corrects it during recording. So, the recording should be down-voted, but we also need to report the sentence.

1. If you choose to report the sentence, it is just recorded into reported.tsv and the recording gets skipped. Some other two volunteers can validate them as the speech is correct.
2. If you choose to down-vote, which is a must for dataset quality, you do not have the option to report the sentence.

Currently, the logical action is to use option #2 for the datasets' sake.

**Describe the solution you'd like**

We need a way to both report and down-vote the sentence. For this:

a) We can have a third button: "No & Report"
b) On the report pop-up page, a checkbox titled "Also invalidate"

are the options that come to mind.

**Comment by robovoice1** [comment]
#3394

**Comment by HarikalarKutusu** [comment]
Oh @robovoice1, I either missed or forgot that one - it has been nearly two years...
This is actually something that is being raised by the community repeatedly as they get confused about what the correct action should be.

**Comment by robovoice1** [comment]
No prob, we.need more upvotes for this request 🙂

**Comment by robovoice1** [comment]
In combination with this request from @HarikalarKutusu ....... 😎

**Comment by robovoice1** [comment]
#2111

**Comment by HarikalarKutusu** [comment]
> 1. Don't skip the clip - require the user to manually make a validation decision after reporting it.

I think this is the correct option.

Otherwise, the points raised by kdavis are relevant.

**Comment by moz-kathyreid** [comment]
I believe this is still a valid Issue, although I couldn't find any reportable sentences in English to test with.

**Comment by HarikalarKutusu** [comment]
Yes, it is still an important issue. It is basically caused by bad text-corpus, mainly coming from bulk-additions where communities do not check every single line multiple times.
